Makefile

```diff
@@ -2,7 +2,7 @@
 .SILENT:
 .PHONY: clean-sparkctl
 
-SPARK_OPERATOR_GOPATH=/go/src/github.com/GoogleCloudPlatform/spark-on-k8s-operator
+SPARK_OPERATOR_GOPATH=/go/src/github.com/bagnaram/spark-on-k8s-operator
```

Comment:
This should probably be GoogleCloudPlatform instead of bagnaram

Reply:
Thank you, I will update this back when I am ready to review